lib/active_force/active_query.rb

```diff
@@ -33,12 +33,6 @@ def limit limit
       limit == 1 ? to_a.first : self
     end
 
-    def where args=nil, *rest
```

Comment:
I think we should look for a solution that doesn't change the roles of Query and ActiveQuery this way.